app/_locales/th/messages.json

```diff
@@ -164,10 +164,29 @@
     "message": "แก้ไขผู้ติดต่อ"
   },
   "endOfFlowMessage1": {
-    "message": "คุณผ่านการทดสอบแล้ว กรุณารับผิดชอบในการรักษา Seed Phrase ให้ปลอดภัย"
+    "message": "คุณผ่านการทดสอบแล้ว โปรดรักษา seedphrase ของคุณให้ปลอดภัย ถือเป็นความรับผิดชอบของคุณ!"
+  },
+  "endOfFlowMessage2": {
```

Comment:
missing: endOfFlowMessage10 ("All Done")

Reply:
I added the missing translation in fce66895e8393a88364f4b070cfebbd506c03d1e.

Thank you.